variables.md

```diff
@@ -1,204 +1,195 @@
+ The `let` keyword was introduced in ES6. The `let` keyword is used to declare a variable which can later be reassigned a different value.
 
- ```
- let meal = 'Enchiladas';
- console.log(meal); // Output: Enchiladas
- meal = 'Burrito';
- console.log(meal); // Output: Burrito
+ ```js
+ let sport = 'Soccer';
+ console.log(sport); // Output: Soccer
+ sport = 'Basketballl';
+ console.log(sport); // Output: Basketball
  ```
 
-#### Similarities of ```var``` and ```let``` keywords
-- Both can be used to declare a variable without assigning the variable a value. In such a case, the variable will be automatically initialized with a value of ```undefined```:
- ```
- let price;
- console.log(price); // Output: undefined
- price = 350;
- console.log(price); // Output: 350
- ```
+#### Similarities of `var` and `let` keywords
+- Both the `var` and `let` can be used to declare a variable without assigning a value to it. In this case, the variable will automatically be initialized with a value of `undefined`:
 
+ ```js
+ let field;
+ console.log(field); // Output: undefined
+ field = "javascript";
+ console.log(field); // Output: javascript
  ```
- var price;
- console.log(price); // Output: undefined
- price = 350;
- console.log(price); // Output: 350
- ```
- If we don’t assign a value to a variable declared using the ```let``` keyword, it automatically has a value of undefined.
-
 
-- From the above similarity, it can be established that the value of both variables can be reassigned.
+ ```js
+ var field;
+ console.log(field); // Output: undefined
+ field = "javascript";
+ console.log(field); // Output: javascript
+ ```
+ 
+From the examples above, it can be established that the value of both variables can be reassigned.
 
 
-#### Disimilarities between ```var``` and ```let``` keywords
+#### Disimilarities between `var` and `let` keywords
 
-the difference between them is that ```var``` is function scoped and ```let``` is block scoped. example:
+the difference between them is that `var` is function scoped and `let` is block scoped. example:
 
-```
-console.log(x);
-var x=5;
-console.log(x); //Output: undefined, 5
+```js
+console.log(num);
+var num=6;
+console.log(num); //Output: undefined, 6
 ```
 
-if an undeclared variable is logged to the console and later declared with ```var``` and assigned a value, it will return undefine and the value assigned to it.
+if an undeclared variable is logged to the console and later declared with `var` and assigned a value, it will return undefine and the value assigned to it, an error will be returned if `let` is used in declaring such variable. example:
 
-an error will be returned if let is used in declaring such variable. example:
 
-
-```
-console.log(x); // output: ReferenceError: x is not defined
-let x=5;
-console.log(x); //Output: 5
+```js
+console.log(num); // output: ReferenceError: x is not defined
+let num=6;
+console.log(num); //Output: 6
```

Comment:
If this code is together, will the code be able to reach the third line even after the ReferenceError?

Reply:
I checked the code and discovered the code won't be able to reach the third line. Then I'll add this to the article